logtape/types.ts

```diff
@@ -0,0 +1,10 @@
+export type * from "./config.ts";
+export type * from "./filesink.jsr.ts";
+export type * from "./filter.ts";
+export type * from "./formatter.ts";
+export type * from "./level.ts";
+export type * from "./logger.ts";
+export type * from "./record.ts";
+export type * as Sink from "./sink.ts";
```

Comment:
Do you need all of these types? 😅

Reply:
For extensibility purposes yes, all the types are required, however, if I'm building in context into logtape then the need decreases a bit but other developers extending on logtape might still need them.